cpp/cmake_modules/FindgRPCAlt.cmake

```diff
@@ -45,6 +47,7 @@ if(GRPCPP_PC_FOUND)
                    HINTS ${GRPCPP_PC_STATIC_LIBRARY_DIRS})
       list(APPEND GRPCPP_LINK_LIBRARIES "${GRPCPP_LIBRARY_${GRPCPP_LIBRARY_NAME}}")
     endforeach()
+    list(APPEND GRPCPP_LINK_LIBRARIES Threads::Threads)
```

Comment:
This is duplicated from L39, should we move it under the if as it is appended to the list in all cases?

Reply:
(I apparently had this open while it was merged ^^ so we can do it in another related PR)

Reply:
Oh, sorry. Could you work on this?